junit-commons/src/main/java/org/junit/gen5/commons/util/Preconditions.java

```diff
@@ -33,10 +30,12 @@ private Preconditions() {
 	 *
 	 * @param object the object to check
 	 * @param message precondition failure message
+	 * @return the passed in object
+	 * @throws NullPointerException if the object is {@code null}
 	 * @see #notNull(Object, Supplier)
 	 */
-	public static void notNull(Object object, String message) throws IllegalArgumentException {
-		notNull(object, () -> message);
+	public static <T> T notNull(T object, String message) throws NullPointerException {
+		return notNull(object, () -> message);
 	}
```

Comment:
That's a nice improvement. I've seen that elsewhere, but I somehow forgot about implementing it like that in JUnit 5.